core/src/main/scala/shapeless/generic.scala

```diff
@@ -295,7 +295,7 @@ trait ReprTypes {
 }
 
 trait CaseClassMacros extends ReprTypes with CaseClassMacrosVersionSpecifics {
-  val c: whitebox.Context
+  val c: blackbox.Context
```

Comment:
😢 

https://github.com/limansky/beanpuree/issues/11
https://github.com/limansky/beanpuree/blob/e86098b9674805966b3b75d62351a61ad1a50b61/src/main/scala/me/limansky/beanpuree/BeanUtils.scala#L21-L22

Reply:
@xuwei-k we've broken binary compatibility for macros in many ways. Unfortunately it's too difficult to maintain. At least the error you get is a compile time error so as soon as the other library which is using macros updates it will work again.